test/common/validate_sso_config.go

```diff
@@ -0,0 +1,72 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	"github.com/headzoo/surf"
+	brow "github.com/headzoo/surf/browser"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestSSOconfig(t TestingTB, ctx *TestingContext) {
+
+	// Validate CPU value requested by SSO
+	t.Log("Validating CPU resources requested for pods")
+	keycloak := &v1alpha1.Keycloak{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: string(integreatlyv1alpha1.ProductRHSSO),
+		},
+	}
+	t.Log("Retrieving Keycloak CR")
+	err := ctx.Client.Get(goctx.TODO(), k8sclient.ObjectKey{Name: keycloak.Name, Namespace: NamespacePrefix + keycloak.Name}, keycloak)
+	if err != nil {
+		t.Fatalf("Couldn't get RHSSO config: %v", err)
+	}
+
+	expected := resource.MustParse("0.65")
+	received := keycloak.Spec.KeycloakDeploymentSpec.Resources.Requests.Cpu()
+	t.Log("Checking requested 650")
+	if !(expected.Equal(*received)) {
+		t.Fatalf("Wrong requested CPU. Expected: %v but got: %v", expected, received)
+	}
+	t.Log("Success")
```

Comment:
"Checking requested 650" could be a major stage in our test, however, I would suggest editing the log at 32 to be more meaningful and remove all of the "t.Log("Success")